assets/js/src/listing/listing.jsx

```diff
@@ -381,7 +381,10 @@ class Listing extends React.Component {
       action: 'bulkAction',
       data,
     }).done(() => {
-      this.getItems();
+      const isEmptyTrashAction = selectedIds === 'all' && params.group === 'trash' && params.action === 'delete';
+      if (!isEmptyTrashAction) {
```

Comment:
I think we need an explanation here. if an engineer comes here after some time they wouldn't understand why do we have this here.